ODIN_II/regression_test/benchmark/task/mixing_optimization/config_file/multconfig.xml

```diff
@@ -0,0 +1,29 @@
+	<config>
+		<verilog_files>
+			<!-- Way of specifying multiple files in a project! -->
+			<verilog_file>regression_test/benchmark/verilog/mixing_optimization/adders_mults.v</verilog_file>
```

Comment:
any chance you can hook onto the regression to build a matrix of the test rather than forcing it through the XML config?